datasets/restaurant.py

```diff
@@ -107,7 +107,7 @@ def extract_order_items(self, message: str) -> list[str]:
 
         items = list()
         for item_dict in response["order"]:
-            if item_dict["is_drink"]:
+            if item_dict["is_drink"] and should_order_drink:
```

Comment:
(reminder) It will probably be better to have separate methods for drinks and food. This change results in the verbatim menu entry being added to the list (see line 116).